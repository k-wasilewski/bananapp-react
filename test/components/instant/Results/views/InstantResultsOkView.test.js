import React from 'react';
import renderer from 'react-test-renderer';
import store from "../../../../../src/redux/store";
import {BrowserRouter} from "react-router-dom";
import {Provider} from "react-redux";
import InstantResultsOkView from "../../../../../src/components/instant/Results/views/InstantResultsOkView";

describe("InstantResultsOkView rendering specification", () => {
    it('InstantResultsOkView is rendered', () => {
        const component = renderer.create(
            <Provider store={store}>
                <BrowserRouter>
                    <InstantResultsOkView />
                </BrowserRouter>
            </Provider>
        );
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});