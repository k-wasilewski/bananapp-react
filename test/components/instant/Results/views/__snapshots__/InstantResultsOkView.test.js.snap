// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InstantResultsOkView rendering specification InstantResultsOkView is rendered 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="your banana"
      className="img"
    />
    <h3>
       Your banana is 
       old, 
    </h3>
    <h3>
       with 
      NaN
      % certainty
    </h3>
    <a
      href="/"
      onClick={[Function]}
    >
      <button
        variant="outlined"
      >
        Back
      </button>
    </a>
  </header>
</div>
`;
